pkgs/applications/science/biology/hisat2/default.nix

```diff
@@ -0,0 +1,51 @@
+{stdenv, fetchFromGitHub, unzip, which, python}:
+
+stdenv.mkDerivation rec {
+  name = "hisat2-${version}";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "infphilo";
```

Comment:
Why do you not get the source from the official release tarball? 

ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/downloads